<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelNbrBonus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de Bonus :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nbrBonusSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelNbrBattle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de Batailles :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="battleSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelNbrPapillon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de Papillons :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="papillonSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelCommandant">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de Commandants :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="commandantSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlien">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre d'Aliens :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="alienSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelBossType">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type de Boss :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="bossComboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>First_Boss</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Second_Boss</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Third_Boss</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelBossCount">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de Boss :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="bossSpinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileName">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom du fichier :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>level_1.json</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Générer le fichier JSON</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
